src/lib/statistics/column_statistics.hpp

```diff
@@ -36,7 +36,7 @@ class ColumnStatistics : public BaseColumnStatistics {
    */
   std::shared_ptr<BaseColumnStatistics> clone() const override;
   FilterByValueEstimate estimate_predicate_with_value(
-      const PredicateCondition predicate_condition, const AllTypeVariant& value,
+      const PredicateCondition predicate_condition, const AllTypeVariant& variant_value,
```

Comment:
We use "value" in other places for instances for AllTypeVariant, I think the name is fine

Reply:
`value` is used in the .cpp once `variant_value` is unpacked. Also mismatching parameter name.